index.js

```diff
@@ -69,32 +69,34 @@ export const box = {
 };
 
 export const step = {
-  stepVertical: 'step-v grid f-grid grid-flow-col gap-x-16',
-  stepVerticalLeft: 'step-v-l',
-  stepVerticalRight: 'step-v-r text-right',
-  stepHorizontal: 'step-h flex-1 grid f-grid gap-y-16 items-center',
+  step: 'group/step',
+  stepVertical: 'group/stepv grid-rows-[20px_auto] grid grid-flow-col gap-x-16',
+  stepVerticalLeft: 'grid-cols-[20px_1fr]',
+  stepVerticalRight: 'grid-cols-[1fr_20px] text-right',
+  stepHorizontal: ' group/steph grid-rows-[auto_20px] grid-cols-[1fr_20px_1fr] flex-1 grid gap-y-16 items-center',
 
-  stepDot:
-    'rounded-full border-2 h-20 w-20 transition-colors duration-300',
+  stepDot: 'rounded-full border-2 h-20 w-20 transition-colors duration-300',
   stepDotVertical: '',
   stepDotVerticalLeft: '',
   stepDotVerticalRight: 'col-start-2',
   stepDotHorizontal: 'row-start-2 justify-self-end',
 
-  stepDotActive: 'border-blue-600 bg-blue-600 text-blue-600',
-  stepDotComplete: 'border-blue-600 bg-blue-600 text-white',
-  stepDotIncomplete: 'border-bluegray-300 bg-white text-white',
+  stepDotActive: 'i-border-$color-step-indicator-handle-border-active i-bg-$color-step-indicator-handle-background-active i-text-$color-step-indicator-handle-background-active',
+  stepDotComplete: 'i-border-$color-step-indicator-handle-border-active i-bg-$color-step-indicator-handle-background-active i-text-$color-step-indicator-icon',
+  stepDotIncomplete: 'i-border-$color-step-indicator-handle-border i-bg-$color-step-indicator-handle-background i-text-$color-step-indicator-icon',
 
-  stepLine: 'step-line transition-colors duration-300',
+  stepLine: 'group-last/stepv:hidden step-line transition-colors duration-300',
```

Comment:
Can we remove `step-line` now? Checking [here](https://github.com/fabric-ds/css/blob/49b07c05686ef132b203026e941bf04dbab6fc7a/src/components/steps.css#L19) and after the `hidden` fix it is probably not needed? 
```suggestion
  stepLine: 'group-last/stepv:hidden transition-colors duration-300',
```

Reply:
Good catch, thanks!